### PR TITLE
FEAT: Support Vicuna-v1.5 and Vicuna-v1.5-16k

### LINKED ISSUE #296: ENH: support vicuna v1.5
### Is your feature request related to a problem? Please describe
Including:
- https://huggingface.co/lmsys/vicuna-7b-v1.5
- https://huggingface.co/lmsys/vicuna-7b-v1.5-16k
- https://huggingface.co/lmsys/vicuna-13b-v1.5
- https://huggingface.co/lmsys/vicuna-13b-v1.5-16k

Please note the 16K models should be in an individual model family.

### PR DESCRIPTION
**Pull request by RayJi01**
Support 7B and 13B models for each model format
Resolves #296

### CODE REVIEW

**Comment by RayJi01** [comment]
Testing for the chat functionality yield good results:

For Vicuna-V1.5-16K 7B 4-bit:
image

For Vicuna-V1.5 7B 4-bit:
image

This ensure that the model yield good result.

File: doc/source/models/builtin/vicuna-v1.5-16k.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- **Model Format:** pytorch
+- **Model Size (in billions):** 7
+- **Quantizations:** 4-bit, 8-bit, none
+- **Model ID:** lmsys/vicuna-7b-v1.5-16k
+
+Execute the following command to launch the model, remember to replace `${quantization}` with your
+chosen quantization method from the options listed above::
+
+   xinference launch --model-name vicuna-v1.5-16k --size-in-billions 7 --model-format pytorch --quantization ${quantization}
+
```
Comment (pangyoki) [inline doc/source/models/builtin/vicuna-v1.5-16k.rst:26]: note that `4-bit quantization is not supported on macOS.`

File: doc/source/models/builtin/vicuna-v1.5-16k.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- **Model Format:** pytorch
+- **Model Size (in billions):** 13
+- **Quantizations:** 4-bit, 8-bit, none
+- **Model ID:** lmsys/vicuna-13b-v1.5-16k
+
+Execute the following command to launch the model, remember to replace `${quantization}` with your
+chosen quantization method from the options listed above::
+
+   xinference launch --model-name vicuna-v1.5-16k --size-in-billions 13 --model-format pytorch --quantization ${quantization}
```
Comment (pangyoki) [inline doc/source/models/builtin/vicuna-v1.5-16k.rst:38]: note that `4-bit quantization is not supported on macOS.`

File: doc/source/models/builtin/vicuna-v1.5-16k.rst [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+===============
+
+- **Model Name:** vicuna-v1.5-16k
+- **Languages:** en
+- **Abilities:** embed, chat
+
+Specifications
+^^^^^^^^^^^^^^
+
+Model Spec (pytorch, 7 Billion)
```
Comment (pangyoki) [inline doc/source/models/builtin/vicuna-v1.5-16k.rst:14]: Model Spec 1

File: doc/source/models/builtin/vicuna-v1.5-16k.rst [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+- **Model Size (in billions):** 7
+- **Quantizations:** 4-bit, 8-bit, none
+- **Model ID:** lmsys/vicuna-7b-v1.5-16k
+
+Execute the following command to launch the model, remember to replace `${quantization}` with your
+chosen quantization method from the options listed above::
+
+   xinference launch --model-name vicuna-v1.5-16k --size-in-billions 7 --model-format pytorch --quantization ${quantization}
+
+Model Spec (pytorch, 13 Billion)
```
Comment (pangyoki) [inline doc/source/models/builtin/vicuna-v1.5-16k.rst:27]: Model Spec 2

File: doc/source/models/builtin/vicuna-v1.5.rst [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+===========
+
+- **Model Name:** vicuna-v1.5
+- **Languages:** en
+- **Abilities:** embed, chat
+
+Specifications
+^^^^^^^^^^^^^^
+
+Model Spec (pytorch, 7 Billion)
```
Comment (pangyoki) [inline doc/source/models/builtin/vicuna-v1.5.rst:14]: Model Spec 1

File: doc/source/models/builtin/vicuna-v1.5.rst [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+Execute the following command to launch the model, remember to replace `${quantization}` with your
+chosen quantization method from the options listed above::
+
+   xinference launch --model-name vicuna-v1.5 --size-in-billions 7 --model-format pytorch --quantization ${quantization}
+
+.. note::
+
+   4-bit quantization is not supported on macOS.
+
+Model Spec (pytorch, 13 Billion)
```
Comment (pangyoki) [inline doc/source/models/builtin/vicuna-v1.5.rst:31]: Model Spec 2

**Review by pangyoki — APPROVED** [review]
LGTM

**Pull request merged**